root/static/scripts/tests/edit/utility/getUpdatedTrackArtists.js

```diff
@@ -122,6 +129,112 @@ test('getUpdatedTrackArtists', function (t) {
         name('C', '3'),
       ],
     },
+    {
+      desc: 'old name but diff. entity',
+      track: [name('A', '1')],
+      oldRel: [name('A', '2')],
+      newRel: [name('B', '3')],
+      want: [name('B', '3')],
```

Comment:
Yep, that's correct. I think I've seen this come about if there was an earlier cleanup that tried to fix an incorrectly-credited release (using A1 rather than A2, in this case) but missed some tracks.